src/components/CreepypastaForm.js

```diff
@@ -0,0 +1,129 @@
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import styled from 'styled-components';
+import { nanoid } from 'nanoid';
+
+export default function CreepypastaFrom({ handleCreateCreepypasta }) {
+  const [wordCount, setWordCount] = useState(0);
+
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  return (
+    <Form
+      aria-label="Create a creepypasta"
+      onSubmit={handleSubmit(creepypasta => onSubmit(creepypasta))}
+    >
+      <Field>
+        <Label htmlFor="title">Title:</Label>
+        <Input
+          {...register('title', {
+            required: 'You need a spooky title!',
+            maxLength: {
+              value: 25,
```

Comment:
Think about adding a counter for the user, so he/she knows how many characters he is allowed to use.
Another idea would be to add `useForm(mode:all)` , which includes the mode `onChange`, so the error message will appear when the user puts in more than 25 characters